src/prefect/server/models/concurrency_limits_v2.py

```diff
@@ -151,14 +157,6 @@ async def update_concurrency_limit(
         else db.ConcurrencyLimitV2.name == name
     )
 
-    values = concurrency_limit.dict(exclude_unset=True)
-    values.update(
-        {
-            "active_slots": active_slots_after_decay(db),
-            "denied_slots": denied_slots_after_decay(db),
-        }
-    )
-
```

Comment:
Removes unused `values` dict. Pickett please let me know if instead this should be used in the SQL stmt below

Reply:
Yeah, this is a weird one. What I was probably thinking is that anytime we update, even outside of aquiring/releasing slots, we should update active/denied slots because they 'leak' over time. I don't think that's super important here, we take this data into account whenever we do the actual acquire/release. Let's just remove it.

Reply:
Sounds good!